qml/Pages/About.qml

```diff
@@ -34,10 +34,18 @@ Page {
     id: page
     property var textAlignment: TextInput.AlignLeft
     SilicaFlickable {
-        RemorsePopup {id: remorse; anchors.top: parent.top}
+        RemorsePopup {
+            id: remorse
+            anchors.top: parent.top
+        }
         anchors.fill: parent
         clip: true
-        contentHeight: header.height + version.height + text.height + links.height + flattr.height + Theme.paddingLarge * 4
+        contentHeight: header.height
+                       + version.height
+                       + text.height
+                       + links.height
+                       + flattr.height
```

Comment:
TODO: remove flattr links. I dunno what happened to flattr, but I never got my money out (since I refused to validate my identity or something)

Reply:
I've heard flattr changed something, which is why Jolla opted out of it...